helm/cloud-provider-vsphere/templates/configmap.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: vsphere-cloud-config
+  name: {{ .Values.config.name | default "cloud-config" }}
```

Comment:
I don't see `.Values.config.name` in the values files.
Do you mean `.Chart.Name` ?

Reply:
I forgot to update `values.yaml`